cvat-ui/src/components/cvat-app.tsx

```diff
@@ -139,7 +139,7 @@ export default class CVATApplication extends React.PureComponent<CVATAppProps> {
                     />
                 ),
                 duration: null,
-                description: error.length > 200 ? '' : error,
+                description: error.length > 200 ? 'Open browser console to get details' : error,
```

Comment:
Need to have an article. Let's use `your` instead.
```suggestion
                description: error.length > 200 ? 'Open your browser console to get details' : error,
```

Reply:
Done